components/Alert.tsx

```diff
@@ -1,13 +1,17 @@
 import classNames from "classnames";
 
-function getClasses(type) {
+type AlertType = "danger" | "warn" | "info";
+
+function getClasses(type: AlertType) {
   return classNames("alert block p-4", {
     "bg-red-100 text-red-500": type === "danger",
     "bg-yellow-100 text-yellow-500": type === "warn",
     "bg-blue-100 text-blue-500": type === "info",
   });
 }
 
-export default function Alert({ type, children }) {
+const Alert: React.FC<{ type: AlertType }> = ({ type, children }) => {
```

Comment:
:megapuke: can this be done while keeping a `function Name()` syntax?

Reply:
Yup! That's just the way I usually do it. Changed it to use interfaces for defining prop types in my latest commit.